### ISSUE TITLE
Motion sensor detecting moving pistons

### ISSUE THREAD

**Issue by kit-la** [Better-than-Adventure/bta-issues#325]
**Is your feature request related to a problem? Please describe.**
I thought i could do a clock with motion sensors and pistons, but realized they don't detect moving blocks in general.

**Describe the solution you'd like**
If a piston moves a block in front of a motion sensor, it will send a signal as if any other entity had.

**Additional context**
It may have some potential for circuit compactability but it is just a suggestion. I imagine it would be somehow resource intensive to implement, specially with lots of moving blocks and motion sensors, so that's a point against it.

**Comment by 12bones12** [comment]
Motion sensor is only for entities, not block updates.

**Comment by Bigjango13** [comment]
> Motion sensor is only for entities, not block updates.

I think this is a suggestion:
> but this is just a suggestion.

**Comment by MinerB4** [comment]
I don't know if i'm right, but when a piston moves the block it's pushing turns into a FallingBlock (i dunno if the piston does either). It might be detecting this as a moving entity.

**Comment by Bigjango13** [comment]
> I don't know if i'm right, but when a piston moves, the block it's pushing turns into a FallingBlock (i dunno if the piston does either). It might be detecting this as a moving entity.
No, it turns it into a TileEntityPiston, which is not an entity (it's more like a chest). The reason it looks like it's moving is because of an animation.
